hedera-node/configuration/dev/genesis-network.json

```diff
@@ -1,54 +0,0 @@
-{
-  "nodeMetadata": [{
-    "rosterEntry": {
```

Comment:
Why did we delete this ? We don't need this anymore

Reply:
This was just a test resource needed before correct _genesis-network.json_ was being generated [here](https://github.com/hashgraph/hedera-services/blob/16839-restart-hapi-tests/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/hedera/utils/WorkingDirUtils.java#L146).